grails-app/services/org/pih/warehouse/core/OrganizationService.groovy

```diff
@@ -17,7 +17,7 @@ class OrganizationService {
     boolean transactional = true
 
 
-    List selectOrganizations(roleTypes) {
+    List selectOrganizations(roleTypes, active = false) {
```

Comment:
It was a bad idea to leave these untyped, so go ahead and add Boolean and a type for the roleType argument.